student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
+    super(age, parent_permission: parent_permission, name: name)
+    @classroom = classroom
```

Comment:
### [ Optional ]
Hi @BranBayou, what if you could have left the super to be like this: `super(age, parent_permission, name)` without reassigning values to the properties?